src/easydiffraction/experiments/experiment.py

```diff
@@ -338,9 +345,9 @@ def show_current_background_type(self):
 
 
 # TODO: Refactor this class to reuse PowderExperiment
-# TODO: This is not a specific experiment, but rather processed data from
-#  PowderExperiment. So, we should think of a better design.
-class PairDistributionFunctionExperiment(BasePowderExperiment):
+# TODO: This is not a specific experiment, but rather processed data
+#  from PowderExperiment. So, we should think of a better design.
+class PairDistrFuncExperiment(BasePowderExperiment):
```

Comment:
We are either descriptive with `PairDistributionFunctionExperiment` or try to save room with something like `PDFExperiment`.
`PairDistrFuncExperiment` looks neither descriptive nor it is short!